src/Form/LoadTheme.php

```diff
@@ -40,26 +40,28 @@ class LoadTheme {
 	private $theme;
 
 	/**
-	 * Form Theme loading handler
+	 * setup form template
 	 *
-	 * @param string $formTheme Theme ID. Add form_theme shortcode argument to load selective form theme.
-	 *
-	 * @global WP_Post $post
+	 * @since 2.7.0
+	 * @param string $formTemplate
+	 * @param int    $formId
```

Comment:
Please document what this defaults to.

Reply:
fixed:https://github.com/impress-org/givewp/compare/b38638d7c2b3...965ab69fb3f6